app/_data/docs_nav_konnect.yml

```
@@ -151,6 +151,8 @@
           url: /dev-portal/applications/enable-app-reg
         - text: Dynamic Client Registration
           items:
+            - text: Overview
+              url: /dev-portal/applications/dynamic-client-registration/overview
```

```suggestion
              url: /dev-portal/applications/dynamic-client-registration/
```
once the file is renamed, can shorten this URL.